_posts/2016-04-22-middle-aged-CMS-glitch.md

```diff
@@ -0,0 +1,97 @@
+---
+
+title: 'MIDDLE AGED CMS GLITCH: TGIF(80)'
+author: Cecily
+layout: blog-post-tgif
+comments: false
+
+---
+
+
+TGIF! – Thank Grohl, Gauss, Glob, Galileo and Science it’s Friday, and again it's time for our weekly reading recommendations, prepared to help you get some good reading time on your weekend. As usually, they're curated for you by [Cecily](https://twitter.com/skeskali)
```

Comment:
s/As usually/As usual 
(I think we need to correct the template because I make this correction every single week)

also swap they're for they’re